### ISSUE TITLE
Port binding between VM host and container not working

### ISSUE THREAD

**Issue by rragundez** [terraform-google-modules/.github#3]
### TL;DR

We've deployed successfully using this module and app that uses port 8080. The port is exposed inside the docker file via 'EXPOSE 8080' and the app is ran via 'CMD run_app_command -p 8080'

Then we SSH into the VM but 'curl localhost:8080' does not return a response from the application. According to the documentation docker by default is binding all host ports. So I would expect this to just work.

In addition, if we stop the container running and manually run 'docker run IMAGE -p 8080:8080' the. The curl command to localhost:8080 returns the response from the containerized application.

### Terraform Configuration

```hcl
-
```

### Terraform Version

```sh
-
```

**Comment by rragundez** [comment]
Seems that we found the bug from our side.
